### ISSUE TITLE
Issue with Colored Dust Particles

### ISSUE THREAD

**Issue by JaydenJinx** [SkriptLang/Skript#2356]
Error shows everything: https://pastebin.com/u4qTmAik

**Comment by JaydenJinx** [comment]
I cannot find correct syntax either, so this is just my assumed "correct".

"play colored dust at player with color 100, 100, 100" does not work.

**Comment by ilsubyeega** [comment]
Please write according to the template
[english.lang#1348](https://github.com/SkriptLang/Skript/blob/2004cf811fbcbbeb9d6c2e2bd0bf12617987683c/src/main/resources/lang/english.lang#L1348) Since its not colored its coloured

**Comment by JaydenJinx** [comment]
"pattern: (%-color%|colo[u]red) dust"

the "u" is optional. in america, we drop the "u"

**Comment by ilsubyeega** [comment]
> 
> 
> "pattern: (%-color%|colo[u]red) dust"
> 
> the "u" is optional. in america, we drop the "u"

oh didnt see oof

show mob spawner flames at the targeted block to the player (works)
play colored dust at the player's location to the player (parsing works, but not showing)

and i cant see the syntax like "with colors"

**Comment by JaydenJinx** [comment]
Yeah... I know there is somewhere though. I just can't find documentation of it. If you don't specify a color, it displays nothing.

**Comment by FranKusmiruk** [comment]
Closing since there's not enough information given and the syntax used isn't the one Skript uses. Please ask for support on the forums or the Discord server before reporting an issue.

**Comment by JaydenJinx** [comment]
I have asked on 3 different Skript-related Discord servers. Nobody has been able to figure it out. I have searched for hours on SkUnity and SpigotMC and found no results. I have tried different syntaxes, most of which throw syntax errors.

EVEN IF this syntax isn't supported, it shouldn't parse without errors. It show throw an error in-game. It doesn't. The error only shows up when executing that line.
